content/docs/config.md

```diff
@@ -47,7 +47,8 @@ jobs:
 ```
 
 The example above generates visual reports in pull requests:
-[![](/img/cml_first_report.png)](https://github.com/iterative/cml_base_case/pull/2)
+![First CML report in GitHub](/img/cml_first_report.png) _From sample
+[increase forest depth](https://github.com/iterative/cml_base_case/pull/2) PR_
```

Comment:
Changed this to link from a caption instead, as currently it looks pretty strange:

![image](https://user-images.githubusercontent.com/1477535/202658956-191b59d4-a9c8-4c7a-a790-51b1792d13a7.png)

> I do admit it would be nice to be able to prevent images from linking to the img src though but let's make this a docs engine request?